### PR TITLE
feat(balance): Rework the scaling of damage and HP with tier

### PR DESCRIPTION
**Pull request by Quantumshark**
**Balance**

This PR implements the change requested in issue #9802 

### Damage
Damage of all T1 weapons is reduced by 20%. This brings DPS:HP ratios a little lower, making combat longer and giving more time to think, plan and implement tactical approaches. This also makes early-game one-shots much less common. For more information on the reasoning, see the discussion on #9802.

Also reduced the hull and toughness of all minables present in human space by 20%. Mining is a little underwhelming late-game; this makes mining more attractive in the post-human portion of the game, while keeping it unchanged in human space.

The weapons of the T1.2 Hai and the T1.5 Remnant and Korath Exiles are unchanged, as combat at those tiers is generally agreed to be fine as it is. In effect, those become the baseline about which everything else is tweaked. 

Increases the damage of the weapons of the T2 Wanderers, Ka'het and Korath Automata by 10%. Combat at those tiers can be excessively slow (as mentioned [here](https://github.com/endless-sky/endless-sky/pull/9851#issuecomment-1961984659), and this reduces the extent to which that is the case.

Increases the damage of T2.5 Heliarch weapons by 20%. They need to fight the Quarg, and the Quarg are going to have a _lot_ of HP, so to stop these battles turning into agonisingly slow slogs a buff is likely to be needed.

### HP
The scaling of HP with tier is linearised, in a manner that mainly affects higher tiers.
Rather than being 1x, 2x, 4x, 8x, 16x etc as you go up through tiers 1, 1.5, 2, 2.5, 3 etc, it's now 1x, 2x, 3x, 4x, 5x.

This resolves the issue of combat at higher tiers being too slow, while still by no stretch trivialising the step between tiers. Going up against high-tier enemies like the Korath Automata is no longer such a difficult thing to do, and the powerful Sestor hulls don't make combat drag on so much.

## Todo
- [x] Decide how to handle weapons at other tiers - what should happen at T1.2? or T0.9?
- [ ] Test campaign balance - this will almost certainly break the pug fights, they may need their shield gen toned down to match.

### CODE REVIEW

File: data/human/weapons.txt [hunk 1]
```diff
@@ -1470,8 +1470,8 @@ outfit "Nuclear Missile"
 		"optical tracking" 1
 		"trigger radius" 30
 		"blast radius" 150
-		"shield damage" 9000
-		"hull damage" 7000
+		"shield damage" 4500
+		"hull damage" 3500
```
Comment (1010todd) [inline data/human/weapons.txt:1474]: Do we really want to change this? Nobody use them as-is.

Comment (Hurleveur) [reply data/human/weapons.txt:1474]: At least increase the blast radius mb?

**Comment by Hurleveur** [comment]
Isnt cutting it in half a bit overkill? Wouldnt 30% or smth be enough?

**Comment by Quantumshark** [comment]
> Isnt cutting it in half a bit overkill? Wouldnt 30% or smth be enough?

It'd need to be reduced by more than this to match T2. This is a reasonable starting point for testing either way. If it turns out to be too much or too little, I can adjust accordingly.

File: data/human/weapons.txt (hunk 1 above)
Comment (Quantumshark) [reply data/human/weapons.txt:1474]: > At least increase the blast radius mb?

I'd rather avoid mixing this up with different changes like that. This PR is for rescaling the tier scaling of damage, if the nuke needs to be buffed relative to other human weapons that should be a separate PR.

**Comment by Hurleveur** [comment]
I dont like slowing down mining honestly, so what about giving lasers minable dmg that corresponds to old hull dmg and not touching the mining laser's mining/prospecting stats?

**Comment by Quantumshark** [comment]
> How does this affect balance with other weapons between T1 and T2? For example, T1.2 Pulse Turrets are basically comparable to Heavy Laser Turrets, but with this change they're nearly twice as good while also being smaller and more efficient.
> 
> It seems to me that this needs to look at all weapons between T1 and T2 in order to maintain tier relationships properly, or we instead just buff T1 ship HP.

I am aware that I'll also need to adjust weapons between T1 and T2, and probably also scale _up_ T2.5 weapons (if current T2 is becoming the baseline). This is a draft PR, just implementing this change for human space and the aliens that appear in the FW campaign, for initial testing.

**Comment by Hurleveur** [comment]
I don't think we should compare the pulse turret to the heavy laser turret, instead it's supposed to be a better quad blaster turret, but in practice its somehow worse dps wise per outfit space (whilst being bigger??)
Hai on hai combat is much slower than human combat already (their medium is the human's heavy and their heavy is tankier, with better shields, whilst their dps is almost worse on average (ion cannon lol) Even I dont use pulse turrets much and I'm a fan of the hai, about any human weapon is better tbh.

If we are to affect it, it should definitely be by a smaller amount or combat will just take ages (and we'll have to rebalance the wand-uhai fights too), and I'd advise leaving that for another PR. Human space is large enough of a scope to test already.

**Comment by Quantumshark** [comment]
> I think that just goes to show that human dps isn't the issue, it's the low HPs of human ships that are making combat times so short.

In human-human combat, these are equivalent in this sense, so there's no way to say that "it's X rather than Y" - it's the ratio that's the issue, and that's a result of both.

Doubling human ships' HP would have the same effect as halving weapon DPS when only considering human-human combat, except that it would further devalue the already-terrible human regen, whereas this makes it more valuable.

As far as whether it's human DPS or human ship HP that's out of line though, it certainly seems to me that human T1 weapons are often far more damaging than the weapons of the T1.2 Hai or the T1.5 Remnant, especially their missiles. So, I really don't think "human dps isn't the issue" is an accurate statement.

> Does anyone think e.g. Korath or Remnant combat times are too short?

Personally, I think T1.5 combat is probably about right, and T2 combat can sometimes get too long (the Sestor fights in particular), so I'd say it might be better to take T1.5 as a baseline for this rather than T2, and make T2 combat slightly faster.

**Comment by Hecter94** [comment]
> Personally, I think T1.5 combat is probably about right, and T2 combat can sometimes get too long (the Sestor fights in particular), so I'd say it might be better to take T1.5 as a baseline for this rather than T2, and make T2 combat slightly faster.

If anything, it makes sense for early-game combat to be slower than later-game combat to give newer players time to get used to it.

File: data/human/weapons.txt [hunk 2]
```diff
@@ -556,8 +556,8 @@ outfit "Particle Cannon"
 		"firing force" 10
 		"firing heat" 120
 		"hit force" 80
-		"shield damage" 65
-		"hull damage" 62
+		"shield damage" 32.5
```
Comment (EjoThims) [inline data/human/weapons.txt:559]: I'm wondering if it might not be better for this one to be half as fast instead of hit half as hard?

Make it the big bad weapon in Human space that still holds up even in and a bit beyond Hai, if you can hit with it, but give it half as many chances to hit in the first place.

This would also further differentiate it from the somewhat competing Proton as well.

I could see this also being better as a follow up refinement, but since it still maintains the change in DPS ratio, I think it's also still appropriate here.

Comment (bene-dictator) [reply data/human/weapons.txt:559]: You mean fire half as fast, and not the projectile going half as fast, right?

Comment (EjoThims) [reply data/human/weapons.txt:559]: Yea

Comment (demolish238) [reply data/human/weapons.txt:559]: Wouldn't that remove it's main advantage being that it staggers making jumping hard?

**Comment by demolish238** [comment]
I don't agree with upping human hp. Currently hp is the only thing that makes sense with the tiers, being a clear increase as you go up. Weapons are the odd one out with human weapons specifically being much better both due to high damage and sheer versatility compared to every other race.
I do think 50% is too much of a decrease and it will be better to eventually make it smaller, but even 10%-20% would be welcome to make human combat last a little longer. Plus currently no other races really need rebalancing if human weapons get a downgrade, since the humans only fight the Pug, so as long as the decrease doesn't make human weapons COMPLETELY inferior to Hai or Remnant weapons (which are both mediocre at best currently and could use the upgrade this would give to them) I don't see why other races would need to be adjusted at all, or at best only need minor adjustments.

File: data/human/weapons.txt (hunk 2 above)
Comment (Hurleveur) [reply data/human/weapons.txt:559]: It's the proton canon you gotta use for that iirc

Comment (bene-dictator) [reply data/human/weapons.txt:559]: Particle cannon is just as good, but it can be easily tested to see if it still stunlocks

**Comment by Hurleveur** [comment]
The reason is that human weapon become way worse if we dont touch anything else - I wouldnt say it's that big of a deal tho, but I did like that most outfits in the game were comparable.
I think the prices of the various human weapons should go down

**Comment by demolish238** [comment]
> you forgot about the kor sestor at the end of wanderer campaign.

The Humans are meant to be massively outgunned by the Sestor, so lowering human weapons damage would actually make that fight make even more sense why the player is needed to run off to Zenith.

> * Pug -- The player only fights them during FW. The only human NPC vs Pug is during FW with assigned escorts

The only case where human weapons being made weaker means the Pugs tier 1 weapons might need adjusting too, otherwise it might make the Pug fights too hard.

> * Exiles -- The player fights them but certainly not with a stock human ship. Balance won't matter much.
> * Any Human NPC vs. Exile NPC only happens in a few earth systems. The humans already get their butts kicked in those systems, so what's that going to look like with weaker weapons? And does it really matter anyway? I doubt the player is sitting around watching.

The Exiles are also meant to massivly outgun humanity, so making the single Exiles that jump into those systems overpower human ships even more only helps show why they loot humanity and why the Syndicate has so few jump drives.

> * Coalition -- Player may or may not fight them. No human NPC vs. Coalition

The Lunarium is meant to be the underdogs, so having them get weaker weapons from the humans would only help that underdog feeling.


Increasing human HP does the opposite of all this, making humanity stronger and making the few aliens they fight have less of an advantage against humanity, meaning the lore on how the Exiles and Sestor are meant to outgun humanity make less sense, rather than lowering weapons damage which only makes that lore more true. My reason increasing HP isn't the way to go, and I know you said you weren't going down that road but just wanted to point this out.

**Comment by Hurleveur** [comment]
Good point with the korath vs human, prob will have the benefit side effect of making that passive farming harder
The missiles of the korath could use balancing but thats a separate issue

**Comment by xX-Dillinger-Xx** [comment]
I'm not picking sides. Both sides of this discussion have very valid points. Both options come with big pros but equally as big cons. The common thread is that using either option messes with the precious tier system, and as pointed out, will cause ripple effects. 

There is another ripple effect not mentioned, with nerfing weapons.  Some players use _higher end human weapons_ in their ship builds even as far as late game. If you nerf **all** human weapons you remove the option of using them later. Would this be a pro or con?

The big question is, what is the real issue? _**human battles are to short**_ I highly doubt players are referring to whats happening between the NPC's. I bet the complaint stems from "I die too quickly". 
How do you fix that issue?
**_What are the options:_** --- I'll start with the ones I can think of. I'm sure I've missed some. 
- decrease DPS (has ripple effect and alters tier)
- increase HP (has ripple effect and alters tier)
- improve human shield generators ( much less ripple effect ) -- nobody uses human gens. beyond FW anyway and if they did, who cares. 
- alter the system arrival points in some human space systems to spread battles out a bit. Maybe, have two different sets of points. One for players and one for NPC. The start of the fight would be delayed. (if anyone wants to discuss this option, I can expand my thoughts)
- alter the asteroids so they can be used as a tactical feature.  (Zitchas idea, I think)
- Reduce NPC fleet sizes in human space a little so the player isn't ganged up on
- add a small buff to the players ships. DPS and/or HP (maybe a non purchasable (i.e. gift) outfit that adds the buff but stays with the player like licenses do.) It  could be removed later in the game. Let's call it `training wheels`.  I highly doubt any player would complain.  When was the last time you read _**"Ah, I'm too  over powered, I wish my ship was weaker"**_

Any of the above ideas could be tweaked to only focus on early game and human space.

**Comment by Hurleveur** [comment]
Even with those changes you still die super quickly to an argosy with missiles (or just so many things in general)
Also this kind of talk is more suited to the issue itself than the PR, I think

**Comment by EjoThims** [comment]
> There is another ripple effect not mentioned, with nerfing weapons.  Some players use _higher end human weapons_ in their ship builds even as far as late game. If you nerf **all** human weapons you remove the option of using them later. Would this be a pro or con?

Personally I see the continued usage of human weapons as a common occurrence anywhere beyond Hai space as an -absolute- sign that human weapons are simply far too strong.

Even if you just desire a specific role/space usage combo that isn't present elsewhere (like particle cannon range and pushback together), you should be sacrificing overall stats (and DPS specifically) to make that build choice.

Especially when so many of the most advanced weapons are both larger -and- restricted behind either story or looting, it is absurd that starting weapons have any significant competitive edge against them.

That this -same change- will also make early farming and bootstrapping via looting more difficult is a very solid plus in my eyes.

File: data/human/weapons.txt (hunk 2 above)
Comment (EjoThims) [reply data/human/weapons.txt:559]: > Wouldn't that remove it's main advantage being that it staggers making jumping hard?

I am okay with it losing best stunlock spot when it is keeping the higher current per shot damage. That it is highest damage and range -and- best stun option in human space all at once is probably overpowered anyway.

**Comment by EjoThims** [comment]
> I think the prices of the various human weapons should go down

Seconding that's this PR should also drop human weapon prices.

Not as much as the lost DPS, but maybe something like 20-30%?

**Comment by demolish238** [comment]
> Only if we want to say that going from T1 to T1.2 means a full doubling in weapon DPS.

As said I'm not sold on halfling the damage, just decreasing them enough that they aren't as effective.

> Coalition civilian ships are already probably going to need buffs to compete with Heliarch ships in the campaign. Halving T1 DPS won't help with that.

Yeah, and I can see that being an issue for those making the coalition campaign, but it isn't much of a reason not to do this change since that campaign isn't even made yet.

I was just pointing out the Coalition because decreasing the weapon damage isn't gonna hurt that campaigns balance much at all, and the two Korath cases, because increasing hp directly makes both of those enemies less powerful against humans, so while currently the balance between humans and the two Korath factions is fine, increasing human ship hp would directly make the Humans weak Korath strong balance go towards humans being decent at fighting the Korath, which would then make rebalancing the Korath around that something we might need to them do which opens a can of worms with many other factions.

File: data/human/weapons.txt [hunk 3]
```diff
@@ -352,10 +352,10 @@ outfit "Mining Laser"
...
 		"firing energy" 1.1
 		"firing heat" 2.1
-		"shield damage" .8
-		"hull damage" 1.4
-		"minable damage" 1.9
-		"prospecting" 1
+		"shield damage" .4
+		"hull damage" .7
+		"minable damage" .95
+		"prospecting" .5
```
Comment (Hurleveur) [inline data/human/weapons.txt:358]:
```suggestion
		"minable damage" 1.9
		"prospecting" 1
```
I dont see why we'd slow mining down (or if we want to do that it shouldnt be in the same PR, I'd argue)

keeping the old values makes this weapon make even more sense as a mining weapon

File: data/human/weapons.txt [hunk 4]
```diff
@@ -383,10 +383,10 @@ outfit "Mining Laser Turret"
...
```
Comment (Hurleveur) [inline data/human/weapons.txt:389]:
```suggestion
		"minable damage" 1.9
		"prospecting" 1
```
same logic

**Review by Hurleveur — COMMENTED** [review]
I think 50% is overkill and threatens game balance too much; if we have to bring it down by this much its notably because of human missiles being overbuffed in comparison to alien ones.

Now dont get me wrong, I agree that we should still bring all weapons down by a %, but no more than 20-30% or they become *too* bad. Then we can look at specific weapons that can cause issues (like the heavy missile or torpedoes and sidewinders that feel to me like they are too good rn)
I personally think its better to nerf it some amount even if we may have to do it some more later than nerf it into the ground.

Finally, I dont think we should be nerfing mining, especially now that we have a weapon dedicated to it. We could even make lasers do the same dmg as before to asteroids, but not nerfing the mining laser would prob be enough.

I'm mainly a code reviewer tho, so take this with a grain of salt

File: data/human/weapons.txt (hunk 1 above)
Comment (Zitchas) [reply data/human/weapons.txt:1474]: I would suggest leaving the nuclear missile stats unchanged. It is supposed to be utterly terrifying with a high chance of one-shotting many human ships.

While I agree this isn't the time or place to make changes to it to make it more effective, I see no problem with merely leaving it untouched. Regarding balancing it versus other human weapons, it really has no real comparable weapon to contrast with, since no other missile is a single shot taking up a gun port. It isn't used much because it's isn't a weapon suitable for fighting with. It is a weapon for intimidation, genocide, and a strategic warfare of the type that happily sacrifices a ship carrying these weapons to take out fleets. While FW, Navy, and Syndicate Admirals all heartily approve of sacrificing a few small spots to take out a fleet, it is of really low utility to the player for obvious reasons.

Tldr. Leave the nuke as is. It is appropriate to leave as is and balancing is not a concern since it isn't really a relevant weapon anyway.

Comment (Hurleveur) [reply data/human/weapons.txt:1474]: Tbf we're trying to slow down combat, and it is the highest dps weapon, and on top of that isnt actually used in battles, besides against the player (it definitely killed me a bunch)

Comment (Arachi-Lover) [reply data/human/weapons.txt:1474]: I too think the nuke should remain unchanged.
Keeping the nuke's damage as is would also by comparison make the Checkmate branch more rewarding and appealing than it currently is, at least in the sense that it'd be a clearly far better option in the human arsenal.

Comment (Zitchas) [reply data/human/weapons.txt:1474]: lol. I suppose another way to put it, in a discussion about slowing down battles, is to point out that the nuclear missile is the unique, solitary human weapon that is supposed to generate battles that are super short, terrifying, and devastatingly deadly. (I mean, all weapons want to do that, but this is the one weapon where story usage and meta dictates that it has actually achieved that goal)

It is, according to our lore, the only weapon in human space so powerful that every single ship in existence (including aliens, somehow) has a built-in warning light in the cockpit to warn the Captain when a ship carrying these weapons enters the system. 

How our ship knows, I don't know. Space is notoriously filled with radiation, so my guess is our FTL LIDAR that runs our minipap and targeting system can recognize a nuclear missile protruding from a gunport.

File: data/human/weapons.txt (hunk 3 above)
Comment (Quantumshark) [reply data/human/weapons.txt]: Opted instead to reduce mineable HP by the same percentage damage was reduced by.

File: data/human/weapons.txt (hunk 2 above)
Comment (Quantumshark) [reply data/human/weapons.txt]: If you want to make the particle cannon deal twice the damage per hit and fire half as fast, that would be a separate change, which could be made with or without this change.

**Review by Hurleveur — COMMENTED** [review]
I like this a lot more than before, buffing wand weapons might be benefitial for the uhai fights, tho idk about the wand-korath fights - it's prob fine, since they're broken already.
I love the idea of not touching the t1.5 stuff and changing things around it.

~~you forgot to nerf gegno stuff~~ (half ironic, idk if we should, but if you're bringing the scope beyond human space idk why we'd leave them out of it)
This does make me wonder, however, didnt we like the speed at which T2 combat was happening? Should we change it all in the same PR? I understand that you did it to keep human dps relatively low in comparison, but isnt 20% enough? I dont have the answer to those questions tbh, just throwing them out there.

**Comment by brendanjones** [comment]
Adding in my 2 cents: I agree with all the changes as they currently stand (especially not increasing human hp, and only nerfing human dps 20%), except for one thing: I wouldn’t increase Kor Mereti weapon dps 10%. Kor Sestor yes, Mereti no. 

The Shield Disruptor Turret combined with the Husk-Slice Turret is exceptionally dangerous. It’s arguably the best weapon combo for taking down T3 ships. I don’t think they need buffing, when a fleet of 10 automata loaded with those turrets makes short work of almost any ship.

File: data/human/weapons.txt (hunk 2 above)
Comment (EjoThims) [reply data/human/weapons.txt]: Twice the damage and half as fast was just because the nerf was 50% at that point, meaning it would keep the per shot damage but still suffer the same DPS drop as other weapons.

Now that this has grown beyond just species' weapons, though, it is indeed much less appropriate here vary how we apply the overall changes for any particular weapon.

**Comment by AvianGeneticist** [comment]
In my opinion, increasing human HP would be a better change than reducing weapon DPS. While this doesn't fit the lore particularly well, I think the gameplay benefit outweighs the narrative complications.

In general, good balance (as in, well-designed balance, not all things being perfectly equal) serves to make a game more interesting by making more options more viable for more players. At the moment, while most human weapons *can* be superceded by alien weapons of similar role (Plasma Cannon -> Fire Lance, Proton Gun -> Annihilator, Particle Cannon -> Mandible Cannon, et cetera), human weapons still have niches where they might be slightly better in, such as the Plasma Cannon having longer range than alien heat weapons or Proton Guns having a decent size, a turreted variant, and good balanced damage (whereas most alien weapons skew a fair bit to one way or the other; typically towards shields).

Reducing damage wouldn't make the sandbox more interesting. Alien weapons are already (generally) better than human weapons (barring a few examples like the Jav Pod comparing well with Sunbeams and whatnot), so nerfing them just serves to further make the hole worse. While I understand the argument of progression, I feel like Regeneration and Energy outfits (and to a lesser extent, Engines) already give plenty of reason to seek out higher tiers, and weapons should be permitted to stay more even. Keeping the weapon sandbox in a state where (almost) everything is relevant all the time only expands the variety of builds that are usable, and encourages further experimentation by mixing weapons the player already has and new ones they've just found (for example, mixing Inhibitors and Particle Cannons for extremely strong ranged performance).

Increasing HP by 20% at the human level wouldn't wreck balance relative to aliens, either. 20% for the biggest human ships is only about 6000 total HP or so if I recall correctly. An appreciable increase, but nothing that'd wreck balance, even relative to Hai-tier ships.

**Comment by Bubadark** [comment]
yes Azure! as an old EV pro when i was a new ES player  i was not able to do any fighting except sniping light warships close to the fence in a Bastion. if i came any closer to system center i would immediately get pig piled in my slow ship. 1v1 fights are not so much the problem its the spawn rate and how much skill it takes to carefully kite something away from the spawns so you can have a somewhat controlled fight with few adds! preventing prey ships from fleeing is a huge problem when a newer player or one who want to take challenging fights even now i tend to just grab an overkill fleet of ships if i want to acquire Inferno turrets or something. 

I am working on a personal mod so all non merchant ships follow the "scans planets" AI then reducing the spawn rate of all those ships by 4x to see what that does. feels internally constant that military and pirate ships would patrol for a bit before moving on, fights should still happen because ships stay in system longer but it should reduce the chaos pig pile of death things so often degenerate into. 
I am not so sure how on board i am with steepening the damage curve i like being able to use weapons from Wanderer, Remnant, Bun, Korath 3x Khet, and Heliarch I even use human to what i feel is best in class effect especially if i want to use economical missile fighters! each have their areas and strengths! Hai are sadly junk outclassed by everyone else.

**Review by TheGiraffe3 — COMMENTED** [review]
There's also the newly-added Successors and Incipias that need to be considered now.
Wouldn't taking the Pug generators down by 20% be enough?

**Comment by Zitchas** [comment]
I'm still of the opinion that dying too fast is not a problem with the weapons, but with how they are distributed and how NPC ships are built.

For instance, the high rate of prevalence of consumable weaponry on pirate ships. They shoot a missile they have spent most of the profit they are going to make from stealing a ship's cargo. And significantly reduced the liklyhood that they are going to get to do any looting at all in the first place. And put themselves in a position where they are now at a loss, so they *have* to continue pressing the attack or today was a big loss of funds. And if they unload *all* their ammo on a target (which most will try to do), then there's pretty much no path to a profitable day for them short of actually capturing the target ship. Pirates using consumables on a regular basis is just a lose-lose-lose proposition for them. And then there is peak pointlesseness in the heavy rocket clipper. Whatever pirate flies that is *obviously* a rich kid playing at being a pirate, since it would be hard to build a ship configuration any less suitable for piracy if they tried.

And then there's the fact that most of the stock ships are currently the result of people tweaking them over the years to be better and more optimized. They're not average all round ships anymore. Which both makes it much harder for players to make meaningful beneficial changes to them, and more deadly. Another lose-lose element. Sure, there should definitely be some optimized ships. But those should be rarer variants, not stock and/or common.

**Comment by Quantumshark** [comment]
> I'm still of the opinion that dying too fast is not a problem with the weapons, but with how they are distributed and how NPC ships are built.

I do agree that the points you raise are also factors in the issue.
But, independently of that, it is also true, objectively, that the ratio of DPS to HP is higher in human space than in other areas of the game, and that that also results in ships being destroyed more quickly in combat. In my mind, that's a problem in itself, and this PR is seeking to resolve it as such.

File: data/coalition/coalition weapons.txt [hunk 5]
```diff
@@ -394,12 +394,11 @@ outfit "ion hail"
 		"lifetime" 40
 		"random lifetime" 4
 		"hit force" 1
-		"shield damage" 12
-		"hull damage" 5
+		"shield damage" 14.4
+		"hull damage" 6
 		"ion damage" .3
 		"scrambling damage" .3
```
Comment (Quantumshark) [inline data/coalition/coalition weapons.txt:406]:
```suggestion


```

**Comment by Arachi-Lover** [comment]
Certified [LinearHPScaling](https://github.com/Arachi-Lover/LinearHPScaling) moment.

As far as the stat changes go, I really like what you've done here. My only gripes would be I think the bigger Korath Exile ships should probably also be toned down HP-wise. I have absolutely not looked at how that would math out on HP per ton, just felt they were missing as they're pretty tanky in comparison to everything else leaving as is.

On terms of stats, I think it'd be neat to have this redefine the Tier balance and make it less of a steep climb. On terms of campaign-balance though, ***merging this PR would make it extremely difficult to complete Wanderers in a single Tier-appropriate ship***. I tested with LinearHPScaling a decent amount of campaign combat, and while FW, Remnant stuff, and isolated mission chains (and also the Mesuket moshpit) remain pretty much the same in terms of balance between forces, `wanderers start` becomes a lot more difficult, as the Wanderers vs Unfettered battles currently rely heavily on the superior HP of Wanderer ships to make up for the disproportionate number of ships on both sides.

While this PR doesn't lower Wanderer HP as much as LinearHPScaling, I still would wager it would render the entire Wanderes vs Unfettered portion of the Wanderer campaign very very difficult. Dunno if rebalancing the fleets for missions in that section of Wanderers is better addressed here or in a separate PR that depends on this one, but it should be kept in mind.

**Comment by Azure3141** [comment]
I would also say human damage should only be reduced by 10% at most, given how long it can take to kill some of the larger ships in a non-HW. This would make early FW fights more difficult than they currently are.

**Comment by Hecter94** [comment]
I mentioned this in a [previous comment](https://github.com/endless-sky/endless-sky/pull/9851#issuecomment-1962064976) However, slow combat in human space and free worlds is a good thing because it gives newcomers to the game, who aren't as familiar with combat, time to react and get used to the overall flow.

**Comment by Azure3141** [comment]
If a player is in a smaller ship, dying in 11 seconds instead of 10 seconds to a Cruiser doesn't really help them at all. But taking an extra 10 seconds to kill said Cruiser just makes it more likely that the player will end up swarmed by the numerous other ships they have to fight.

**Comment by Azure3141** [comment]
I would still prefer to leave T1 dps and regen alone and just scale T2.5 weapons higher. Of the T2 weapons, only missiles/secondaries need buffs, and that's more about their total damage potential than anything else.

**Comment by Azure3141** [comment]
Would it be possible to split the changes to HP scaling and the changes to weapons DPS + shield regeneration into separate PRs?

**Comment by Quantumshark** [comment]
> If a player is in a smaller ship, dying in 11 seconds instead of 10 seconds to a Cruiser doesn't really help them at all

No, but having 5 seconds rather than 3 seconds to react when a hostile ship opens fire on you really _does_ help, especially if you take maybe 1-2 seconds to process the situation and react to it. Slowing combat down a) helps to offset the advantage the AI gains from its instantaneous reaction times and constant perfect awareness, and b) helps the player a lot in any of the many cases where they're trying to survive, rather than to win a fight - and if they are trying to win a fight, the fact that it affects both sides equally makes it as likely to be a help as a hindrance.

And, in terms of the specifics of large-vs-small matchups, I strongly suspect smaller ships will be the ones to benefit more from having a little more time to actually make use of their relative advantage in mobility rather than just losing because their relative disadvantage in HP leads to them dying before they have the chance.

> Would it be possible to split the changes to HP scaling and the changes to weapons DPS + shield regeneration into separate PRs?

It would certainly be possible - even easy - to do so; the two sets of changes exist largely independently in the data. But it wouldn't be _helpful_, since looking at only one of the two parts and not the other wouldn't give you an accurate picture of what the effects on gameplay of the DPS:HP ratio at any given tier actually were.

**Comment by Quantumshark** [comment]
This also affects the economy side of combat in a few ways.
Within human space, it's easier to disable rather than destroy a ship, but harder to disable a ship before it escapes. This makes hit force more important for making money from combat, while the instant switch-off of beams is less important. That's probably a good thing, because it evens out the effectiveness of different weapon types when trying to make money a bit. Missiles benefit quite a bit from this change, but it also now costs more to buy ammo for the same damage dealt (as you use more of it), so the change there isn't that big overall either.
From experience, the extra time the enemy has to get away seems to outweigh the value of the fact they're less likely to be destroyed, so it's now harder to make money from combat against pirates than it used to be.
